v03_pipeline/lib/tasks/reference_data/update_variant_annotations_table_with_updated_reference_dataset.py

```diff
@@ -0,0 +1,62 @@
+from typing import ClassVar
+
+import hail as hl
+import luigi
+
+from v03_pipeline.lib.model import ReferenceDatasetCollection
+from v03_pipeline.lib.reference_data.compare_globals import (
+    Globals,
+    get_datasets_to_update,
+)
+from v03_pipeline.lib.tasks.base.base_variant_annotations_table import (
+    BaseVariantAnnotationsTableTask,
+)
+
+
+class UpdateVariantAnnotationsTableWithUpdatedReferenceDataset(
+    BaseVariantAnnotationsTableTask,
+):
+    rdc = luigi.EnumParameter(enum=ReferenceDatasetCollection)
+    _datasets_to_update: ClassVar[list[str]] = []
+
+    def complete(self) -> bool:
+        self._datasets_to_update.clear()
+
+        if not super().complete():
+            self._datasets_to_update.extend(
```

Comment:
This might be cleaner w/ just setting to the list.  It should always be empty at this point right?

```
self._datasets_to_update = self.rdc.datasets(self.dataset_type)
```

Reply:
we still need `extend` now that the task checks multiple rdcs

Reply:
What about on line 34?